daemon/web/pages/containers/Containers.js

```diff
@@ -24,13 +26,19 @@ class ContainersWrapper extends React.Component {
     return (
       <div>
         <IconHeader type="containers" title="/inertia-deploy-test_dev_1" />
-        <div className="containerInfo" />
+        <div className="containerInfo" >
+          <Status title="Status:" status="Active" />
+          <h3>Last Updated:</h3>
```

Comment:
it might be useful to have two `Status` component types, thoughts @asitu @PiggySpeed ?

```js
<StatusBadge title="Status:" status="Active" />
<StatusText title="Last Updated:" status={this.props.dateUpdated} />
```

That can wait for another PR though, if you want to open up a new issue @asitu (opening up issues is fun)